src/docs/product/session-replay/index.mdx

```diff
@@ -26,7 +26,7 @@ The start of a replay recording can be triggered by:
 
 The end of a replay recording can be triggered by:
 
-- User inactivity within the tab/page that’s being recorded. (If a user doesn’t click or navigate around the site for more than 5 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)
+- User inactivity within the tab/page that’s being recorded. (If a user doesn’t click or navigate around the site for more than 15 minutes. Mouse scrolls, mouse movements, and keyboard events don’t currently qualify as activity.)
```

Comment:
do we need to qualify this by SDK version?

Reply:
No strong feelings on this, but I think people going here looking for these docs can be assumed to be on the latest version, likely. And this in particular is not _that_ critical that it will break somebody or something like that if they _are_ on an older version.